benchmark/benchmarks.dart

```diff
@@ -103,60 +190,30 @@ class SearchBenchmark1 extends RTreeBenchmarkBase {
   }
 
   void setup() {
-    tree = RTree(BRANCH_FACTOR);
+    size = sqrt(totalItems).ceil();
+    tree = RTree(branchFactor);
 
+    var datum = <RTreeDatum<String>>[];
     for (int i = 0; i < 10; i++) {
       for (int j = 0; j < 50; j++) {
         Rectangle rect = Rectangle(i, j, 1, 1);
-        tree.insert(RTreeDatum<String>(rect, 'item1'));
-        tree.insert(RTreeDatum<String>(rect, 'item2'));
-        tree.insert(RTreeDatum<String>(rect, 'item3'));
-        tree.insert(RTreeDatum<String>(rect, 'item4'));
-        tree.insert(RTreeDatum<String>(rect, 'item5'));
-        tree.insert(RTreeDatum<String>(rect, 'item6'));
-        tree.insert(RTreeDatum<String>(rect, 'item7'));
-        tree.insert(RTreeDatum<String>(rect, 'item8'));
-        tree.insert(RTreeDatum<String>(rect, 'item9'));
-        tree.insert(RTreeDatum<String>(rect, 'item10'));
+        datum.add(RTreeDatum<String>(rect, 'item1'));
+        datum.add(RTreeDatum<String>(rect, 'item2'));
+        datum.add(RTreeDatum<String>(rect, 'item3'));
+        datum.add(RTreeDatum<String>(rect, 'item4'));
+        datum.add(RTreeDatum<String>(rect, 'item5'));
+        datum.add(RTreeDatum<String>(rect, 'item6'));
+        datum.add(RTreeDatum<String>(rect, 'item7'));
+        datum.add(RTreeDatum<String>(rect, 'item8'));
+        datum.add(RTreeDatum<String>(rect, 'item9'));
+        datum.add(RTreeDatum<String>(rect, 'item10'));
       }
     }
-  }
-
-  void teardown() {}
-}
-
-class SearchBenchmark2 extends RTreeBenchmarkBase {
```

Comment:
the purpose of this benchmark compared to the other didn't seem very clear to me, let me know if I am missing something and there's a good reason to keep it around